src/rabbit.erl

```diff
@@ -40,6 +40,13 @@
                     {requires,    pre_boot},
                     {enables,     external_infrastructure}]}).
 
+%% rabbit_alarm currently starts memory and disk space monitors
+-rabbit_boot_step({rabbit_alarm,
+                   [{description, "alarm handler"},
+                    {mfa,         {rabbit_alarm, start, []}},
+                    {requires,    pre_boot},
```

Comment:
how does moving this earlier in the boot process interacts with the use of `rabbit_log` here: https://github.com/rabbitmq/rabbitmq-server/blob/master/src/rabbit_alarm.erl#L212 ?